<launch>
    <node pkg="compressed_pointcloud_transport" name="cloud_compressor" type="cloud_compressor" respawn="false" output="screen">

      <!-- for a full list of profiles see: /io/include/pcl/compression/compression_profiles.h -->
      <param name="compression_profile" value="LOW_RES_ONLINE_COMPRESSION_WITH_COLOR" />
      <remap from="input" to="/camera/rgb/points" />
      <remap from="output" to="/compressed_cloud" />
    </node>
</launch>     
